src/main/java/com/lucidworks/spark/SolrRDD.java

```diff
@@ -675,14 +853,14 @@ public StructType getQuerySchema(SolrQuery query) throws Exception {
     }
 
     Collections.sort(values);
-    final Map<String,Double> labelMap = new HashMap<>();
+    final HashMap<String, Double> labelMap = new HashMap<String, Double>();
```

Comment:
@thelabdude I had to adjust this to get this to compile after rebasing against master.